src/_variables.scss

```diff
@@ -183,7 +183,7 @@ $layout-drawer-border-color: unquote("rgb(#{$palette-grey-300})") !default;
 $layout-text-color: unquote("rgb(#{$palette-grey-800})") !default;
 $layout-drawer-navigation-color: #757575 !default;
 $layout-drawer-navigation-link-active-background: unquote("rgb(#{$color-light-contrast})") !default;
-$layout-drawer-navigation-link-active-color: unquote("rgb(#{$color-primary})") !default;
+$layout-drawer-navigation-link-active-color: unquote("rgb(#{$palette-grey-300})") !default;
```

Comment:
This is MDL version of a hard-coded color :smile: At least put it in a variable, but I’m not sure grey-300 will work with all of the theme color combinations, or will it?